backend/middlewares/validators/lessonValidators.ts

```diff
@@ -27,9 +28,44 @@ export const lessonRequestDtoValidator = async (
   if (!validatePrimitive(body.image, "string")) {
     return res.status(400).send(getApiValidationError("image", "string"));
   }
-  // TO DO: Write better validator for content
+  // content validation
+  if(!validateContentList(body.content)){
+    return res.status(400).send("There was an error in the lesson content.")
+  }
   if (typeof body.content !== "object") {
     return res.status(400).send("There was an error");
   }
   return next();
 };
+
+
+export const validateContent = (content: ContentBlock): boolean => {
+  if (content.type === undefined || content.type === null) return false;
```

Comment:
typescript question -- would `if (!content.type) return false;` work? Pretty sure the falsy values (including undefined, null) are all not expected values

Reply:
same sort of question for the validation @ ln48, ln52, ln56 but I guess maybe an empty string at `text.content.text` could be accepted